low-level/solution/lr2/server/health.py

```diff
@@ -680,23 +680,31 @@ def get_disks_info(self):
             disk_health = self.get_health_template(uid, True)
             health_data = disk.get_health()
             health = "OK" if (health_data['SMART_health'] == "PASSED") else "Fault"
-            self.set_health_data(disk_health, health, specifics=[{"SMART": health_data}])
+            serial_no = disk.id.split("-")[-1] if disk.id else "NA"
+            disk_health.update({"serial_no": serial_no})
+            self.set_health_data(disk_health, health, specifics=[{
+                "SMART": health_data}])
             disks.append(disk_health)
         logger.debug(self.log.svc_log(
             f"Disk Health Data:{disks}"))
+        disks = ServerResourceMap._sort_list_of_dict_by_kv(
+            disks, "serial_no", key_remove = True)
```

Comment:
why to remove the key, its useful data right? also if kv removed, with what we will show as sorted upon, in generated data

Reply:
> why to remove the key, its useful data right?

"serial_number" is manifest information. and here we are keeping only health information.
> also if kv removed, with what we will show as sorted upon, in generated data.

anyway, I agree with the above point. so keeping this information in health also.
Done